azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -124,6 +129,30 @@ public void testBasicRun() throws IOException {
         .assertEvents(EventType.JOB_STARTED, EventType.JOB_STATUS_CHANGED, EventType.JOB_FINISHED);
   }
 
+  private void checkRequiredJobProperties(JobRunner runner, File logFile) {
+    Field jobField = null;
+    try {
+      jobField = runner.getClass().getDeclaredField("job");
+    } catch (NoSuchFieldException e) {
+      Assert.fail("'job' field not found");
+      e.printStackTrace();
+    }
+    jobField.setAccessible(true);
+    InteractiveTestJob job = null;
+    try {
+      job = (InteractiveTestJob) jobField.get(runner);
+    } catch (IllegalAccessException e) {
+      Assert.fail("'job' field not accessible");
+      e.printStackTrace();
+    }
+    Props jobProps = job.getJobProps();
+    Assert.assertNotNull("Log file path not found in properties",
```

Comment:
You might not need this `assertNotNull` check since it will be covered in the `assertEquals` check later.

Reply:
correct